### ISSUE TITLE
Unable to recreate Timit dataset results using wav2vec pretrained model

### ISSUE THREAD

**Issue by paulista5** [huggingface/blog#98]
Hi,

I am unable to recreate the results reported in your blogpost https://huggingface.co/blog/fine-tune-wav2vec2-english on the timit dataset. I just ran the notebook as it is on google-colab, but got very different results. I am attaching a screenshot of the training and validation results. The results are completely different from the ones reported in the blog post. Can u please let m know the issue here? and how we can recreate the results?
Thanks
<img width="1147" alt="Screenshot 2021-04-07 at 8 05 37 PM" src="https://user-images.githubusercontent.com/10824163/113906043-70c80000-97f1-11eb-8bbc-d1d329d5c7c6.png">

**Comment by imtiaz3990** [comment]
hi i am facing the same issue as @paulista5 . I am doing the same exercise on custom data , still facing the same WER 1.0 can any you @patrickvonplaten please help out here ?

**Comment by patrickvonplaten** [comment]
Hey @paulista5, 

Many people have tried to run the notebook and it seemed to work fine - did you change any settings? Also to me it looks like your model is heavily overfitting -> maybe you should add some weight decay and reduce the learning rate

**Comment by sourabharsh** [comment]
Hi, I am also facing the same issue. The WER is fixed at 1 throughout the training.

**Comment by sourabharsh** [comment]
Also, the timit dataset seems to have identical samples in your notebook. 
<img width="1440" alt="Screen Shot 2021-06-26 at 11 45 10 PM" src="https://user-images.githubusercontent.com/5606701/123522066-9a9a1000-d6d8-11eb-9286-3803b4669688.png">

**Comment by patrickvonplaten** [comment]
I think this happened with an old version of `datasets`, could you try updating the datasets library?

**Comment by ahmedlone127** [comment]
![image](https://user-images.githubusercontent.com/66001253/126315332-3d98ef9a-4540-4e88-aabe-1d60ff955fd7.png)

can anyone help me out with this?

**Comment by CantaoSu** [comment]
> Hey @paulista5,
> 
> Many people have tried to run the notebook and it seemed to work fine - did you change any settings? Also to me it looks like your model is heavily overfitting -> maybe you should add some weight decay and reduce the learning rate

@paulista5 , Hi I didn't change anything but also achieved the WER at 1.0 constantly from the very beginning to the end. It turns out the same with 1.0 WER when I tried the other tutorial https://huggingface.co/docs/transformers/tasks/asr 
Could you please solve this issue? It's very important to my thesis.
